src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java

```diff
@@ -92,6 +95,7 @@ public class TarantoolCartridgeContainer extends GenericContainer<TarantoolCartr
     private static final String VSHARD_BOOTSTRAP_COMMAND = "return require('cartridge').admin_bootstrap_vshard()";
     private static final String SCRIPT_RESOURCE_DIRECTORY = "";
     private static final String INSTANCE_DIR = "/app";
+    private static final String REPLICASETS_FILE = "cartridge/replicasets.yml";
```

Comment:
Hardcode is not suitable here. replicasets.yml can be passed as a command line parameter to cartridge cli (see ```cartridge replicasets setup --help```) . Please look at instances.yml processing for example.

Also we already have a ctor parameter ```topologyConfigurationFile```. We can detect the type of file and call lua or yml configuration strategy

Reply:
Did refactoring